### DISCUSSION TITLE
Testimonial

### DISCUSSION THREAD

**Discussion by harinij** [AntlerVC/firetable#697] [Announcement]
If your company or organization is using Rowy, we want to welcome you to leave feedback and share how you are using Rowy.

We are grateful for your replies and they will be a great source of continued motivation for the Rowy team.

Copy-paste this format and reply in the comments.
- **Company/Organization name**:
- **What project or product you are building**:
- **Any links**:
- **Comments:**

**Comment by davhad** [comment]
- Superscout
- Superscout app, an app for VC scouts.
- https://superscout.co
- Been using Rowy to make it easier to manage our Firebase data. It's a great idea and fills a big need for Firebase developers. Wish I had it on previous projects!

**Comment by MatMorris** [comment]
- Freelance, Mat Morris
- Multiple including St Bartholomew’s and Foodbank flow manager
- https://apps.apple.com/gb/app/st-bartholomews/id1631388831
- Makes it easier to share the underlying database with less technical users and allows them to add new data with less support to their applications.

**Comment by alikibao** [comment]
- Freelancer at Afrodynamics
- Multiple projects for clients, from online auction apps, CRMs and ERPs
- I can only share one publicly: https://michaeljames.co.za/
- Rowy allowed me to truly leverage the power of Firebase with limited to no coding experience. I use it not only to manage Firebase data but I am really enjoying the use of Rowy to easily deploy and manage Cloud Functions.

**Comment by tdDeveloperSix** [comment]
- CEO/Developer af talkDigitalX
- Information apps to municipalities
- link to one: https://infotavle.taldigital.dk/signalbakken
- ROWY has enabled med to do all the API backend, allowing me to save money and time. I am not that proficient a developer, so it really helps me a lot. Go ROWY!!!

**Comment by Olmatro** [comment]
* Educator at [Institut des Arts de Diffusion](https://www.iad-arts.be) ;
* Cinematography education (Color Grading and Workflows) ;
* Link to Educators and me : https://www.iad-arts.be/equipe/p3
* ROWY seems to be the best backend for my CMS and Mobile Web Apps that I'm developing for my Students across my workshops  and lessons ;
* [Rowy Bridges](https://www.youtube.com/watch?v=zZxK9m2O_LI) from Airtable are easing a big lot migration to rowy.io ;
* Google Cloud / Firebase platform is a robust industrial standard ;
* Low-code javascript is my preference although I'd already experienced some Python ;
* Rowy's support https://github.com/rowyio/rowy/discussions is interesting especially since questions are followed up and sometimes solved by Rowy's developers themselves.

**Comment by yonghanlin** [comment]
- Student Researcher at [NYCU](https://en.nycu.edu.tw/) in Taiwan
- Research project for improving human-notification interaction
- My personal information: https://scod0401.github.io/
- Rowy is helping a lot with my project by providing easy management of my Firebase data, I really enjoy using it, highly recommended!

**Comment by LWIdevteam** [comment]
Excited to jump into Rowy and connect it up to our Firebase database! A pretty sweet tool.

**Comment by azrilamil** [comment]
Company/Organization name: Universiti Selangor, Malaysia

What project or product you are building: We're working on developing an Outcome Based Education (OBE) back-end for our students and staff.

Any links: [Academic Staff Engineering - FELS](https://fels.unisel.edu.my/training_2022/academician-staff-engineering/)

Comments: As an educator at Universiti Selangor, Malaysia, I've found Rowy to be a game-changer in managing our Outcome Based Education (OBE) back end. Shifting the DB from Excel to Rowy, our work has become considerably faster and simpler, significantly improving our efficiency. It has played a crucial role in streamlining our OBE project, facilitating better collaboration, and reducing errors. Highly recommend!

**Comment by chaseaustin42** [comment]
Company/Organization name: Poiz

What project or product you are building: Mental performance training app for athletes
Any links: https://getpoiz.com/
Comments: As non-technical founder, Rowy makes it easy for us to view and work with our data. If you can work with spreadsheets, you can work with Rowy

**Comment by polingsays** [comment]
Company/Organization name: [Poling - STUDENT]

What project or product you are building: Apps for school use and personal portfolio
Any links: https://polingsays.com
Comments: As a student, Rowy provides a lot of easy tools to let me build apps without professional coding skills, and helped me to generate more speed on building backends.

**Comment by Laiona** [comment]
Company/Organization name: Student (BMSTU)
What project or product you are building: apps for a personal portfolio and student projects
Any links: https://github.com/Laiona
Comments: Rowy will greatly speed up backend development for my projects, as well as facilitate interaction with Firebase via FlutterFlow

**Comment by adambeauferris** [comment]
`Institution: `
Student @ University of Calgary (Alberta, Canada)
`Working on:`
Various personal projects
`Comments:`
Rowy enables me to sidestep the need for deploying a complete code backend solution. Thus far, it has proven to be an excellent and potent tool for this purpose.

**Comment by Mucreth** [comment]
Organization name: STUDENT :)

What project or product you are building: 
Apps for school use and personal portfolio
Any links: 
https://github.com/Mucreth
Comments: 
Rowy offers many easy tools that allow me to build applications without professional coding skills. It will greatly speed up backend development for my projects as well as facilitate interaction with Firebase through FlutterFlow,

**Comment by shapathdas** [comment]
Company/Organization name: Student at Tel Aviv University
What project or product you are building: As a solo founder I am trying to build a decentralized AI social platform, where personal AI can mimic their owners on social media.
Any links: https://toreet.com/
Comments: I have seen many of Rowy's tutorials online and am now excited to use Rowy to speed up my development process as a solo student founder.

**Comment by pierrecorbay** [comment]
Company/Organization name: Student at Efrei Paris
What project or product you are building: A mobile app to do homework with AI
Any links: not yet :/
Comments: I use FlutterFlow and have seen many videos where Rowy seems super useful and easy to integrate with Firebase & FlutterFlow.

**Comment by ryan54321** [comment]
Company/Organization name: Happy-Retired
What project or product you are building: building a data base
Comments: very useful and user friendly
